batch7crud-roll3/src/main/java/com/lftechnology/batch7crud/filter/ExceptionFilter.java

```diff
@@ -0,0 +1,39 @@
+package com.lftechnology.batch7crud.filter;
+
+import java.io.IOException;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletResponse;
+
+import com.lftechnology.batch7crud.constant.CommonConstant;
+
+public class ExceptionFilter implements Filter {
+
+  @Override
+  public void destroy() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1186)